app/utils/userAgentInfo.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import UAParser from 'ua-parser-js';
+
+export default class UserAgentInfo {
+  /** Object structured like this:
+    * Refer: https://www.npmjs.com/package/ua-parser-js
```

Comment:
This comment seems a little excessive. Maybe just a link to the NPM package is sufficient if we can't get types for the library.